sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/kafka/AzureKafkaConfigurationUtils.java

```diff
@@ -49,7 +61,23 @@ public final class AzureKafkaAutoconfigurationUtils {
         KAFKA_OAUTH_CONFIGS = Collections.unmodifiableMap(configs);
     }
 
-    private AzureKafkaAutoconfigurationUtils() {
+    private AzureKafkaConfigurationUtils() {
+    }
+
+    public static void configureKafkaUserAgent() {
```

Comment:
We'd better add some Javadocs for this method, to let people know that it's idempotent.